js/script.js

```diff
@@ -0,0 +1,38 @@
+const tweetLink = "https://twitter.com/intent/tweet?text=";
+const quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+
+function getQuote() {
+    fetch(quoteUrl, { cache: "no-store" })
+        .then(function(resp) {
+            return resp.json();
+        })
+        .then(createTweet);
+}
+
+function createTweet(input) {
+    const data = input[0];
```

Comment:
before you assign first element from response you should be sure that response is an array with at least one element - please check it before

Reply:
done
if (input.length > 0) {
 ///code
}